.circleci/config.yml

```diff
@@ -1002,25 +1002,6 @@ jobs:
                       else
                         echo "# --->>> THIS IS A DRY RUN"
                       fi;
-            - when:
-                  condition:
-                      not: << pipeline.parameters.dry_run>>
-                  steps:
-                      - slack/notify:
-                            channel: C02JENTV2AX
-                            event: pass
-                            custom: |
-                                {
-                                  "blocks": [
-                                    {
-                                      "type": "section",
-                                      "text": {
-                                        "type": "mrkdwn",
-                                        "text": "✅ APIM repository has been tagged.\nYou can continue the release by running: `npm run package_zip -- --version=<< pipeline.parameters.graviteeio_version >>`"
-                                      }
```

Comment:
why deleting this notification ?

Reply:
When we did the release and checked if any update is necessary on the process or the docs, we estimated it's not really necessary for the moment. 🤷🏼 

This is the only one step which do this kind notification and everything is really well described in the release readme. It seems that when this notification has been done it was more a notification test.